cmd/run.go

```diff
@@ -243,6 +245,7 @@ func init() {
 	RunCmd.Flags().StringVarP(&logLevel, "log-level", "", "info", "Sets the log verbosity. Valid values are: debug, info, warn, error, fatal, or panic. Default is info")
 	RunCmd.Flags().IntVarP(&maxConcurrency, "max-concurrency", "", -1, "controls the concurrency level of the app. Default is unlimited")
 	RunCmd.Flags().StringVarP(&protocol, "protocol", "", "http", "tells Dapr to use HTTP or gRPC to talk to the app. Default is http")
+	RunCmd.Flags().StringVarP(&componentsPath, "components-path", "", "", "Path for components directory.")
```

Comment:
It would be good to indicate the default here.  Eg append "Default is ./components"

Reply:
Makes sense! I've made the change 👍 Thanks for the suggestion 🙂

Reply:
I actually just meant as a text comment in the 5th arg.  ie what you had but
```
RunCmd.Flags().StringVarP(&componentsPath, "components-path", "", "", "Path for components directory.  Default is ./components.")
```
I think putting a default value with a slash in the 4th arg may not work on some OSes.

Reply:
My bad! 🤦 Made the change now 👍 Thanks 🙂